CONTRIBUTING.md

```diff
@@ -1,80 +1,141 @@
     ```
 
-    Typically, `script-name` will be something like "Steam", "Online" or "v1.2".
-    
-    Even if the application name contains ®, ™ or the like, you should not use it in the folder name.
-
-5. Fill the files:
-    * [script.js](https://phoenicisorg.github.io/scripts/Develop/script-js/): actual installation script
-    * [script.json](https://phoenicisorg.github.io/scripts/Develop/script-json/): describes the installation script
-        ```json
-        {
-		"scriptName": "Online",
-		"compatibleOperatingSystems": ["MACOSX", "LINUX"],
-		"testingOperatingSystems": [],
-		"free": true,
-		"requiresPatch": false
-        }
-        ```
-    * main.png: application icon (400px x 300px)
-    * [application.json](https://phoenicisorg.github.io/scripts/Develop/application-json/): describes the application
-        ```json
-        {
- 		"name":	"Steam",
-		"description": "Application description"
-        }
-        ```
-  
+Q5. All installers has to work as expected to be approved in official and non-testing repository, if the proposed change is not applicable to be accepted in a repository comments in code are accepted otherwise the installers should be comment-free.
+
+Q999. Everyone has to follow set of rules below
+
+#### Code of Contribution
+These set of rules ensures quality of provided contribution making their merge into master more effective and possible without sacrificing the quality of provided code and contribution.
+
+
+C1. **Trolling is not tollerated**
+*Purposefully participation only to aggrevate other parties of phoenicis is not tollerated and will result in a ban for six hours times two if the issue presist after the ban.*
+
+C2. **Spamming is not tollerated**
+*Purposefully posting content that is not relevant to the project and/or propagates NSFW/malicious content without person's consciousness is not tolerated and will result in a ban for 1 month
+
+If person's account was hijacked he/she can contact team in PhoenicisOrg/PublicRelation and requst unban assuming that person did neccesary procautions to prevent such outcome in the future.*
+
+C3. **Approach the contribution open-minded**
+*Purpose of this set of rules is to ensure code quality including the informations and knowledge that the contributors have. That's why you are required consider the posibility that you are wrong if you are proven to be wrong from verified source.*
+
+C4. **Challenge other members if you have a valid reason**
+*If you think that 'something' in phoenicis project can be improved and you have a verified proof to support your claim, then you have the right to challenge other members of the project.
+
+Valid reason is understand as verified proof meaning that the proof was confirmed by a truested source or is based on valid research.*
+
+C5. **Nothing is permanent**
+*Everyone has the right to suggest changes to the project assuming they have a valid proof to support their reasoning.*
+
+C6. **European Copyright Directive**
```

Comment:
Suggestion: European Copyright Directive

Should be reviewed to prepare for New European Copyright Directive.